java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -779,6 +779,30 @@ public ColumnVector isNull() {
     }
   }
 
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
+   * TRUE for any entry that is Nan, and FALSE otherwise
+   *
+   * @return - Boolean vector
+   */
+  public ColumnVector isNan() {
+    try (DevicePrediction prediction = new DevicePrediction(predictSizeFor(DType.BOOL8),"isNan")) {
+      return new ColumnVector(isNanNative(offHeap.cudfColumnHandle.getViewHandle()));
+    }
+  }
+
+  /**
+   * Returns a Boolean vector with the same number of rows as this instance, that has
+   * TRUE for any entry that is not Nan, and FALSE otherwise
```

Comment:
Here too what happens is the input value is null?  What about if it is not a double or float?